extensions/amp-carousel/0.1/test/test-slides.js

```diff
@@ -382,8 +382,8 @@ describe('Slides functional', () => {
         pos: 0.6,
         min: 0,
         max: 0,
-        prevTr: prevTr,
-        nextTr: nextTr,
+        prevTr,
```

Comment:
so we're ok with the shorthand being anywhere? i thought from the conversation here that they should always be on top